packages/ag-grid-theme/stories/examples/CellDropdownEditor.tsx

```diff
@@ -37,7 +37,7 @@ const CellDropdownEditor = (props: AgGridReactProps) => {
   ]);
 
   const { isGridReady, agGridProps, containerProps, api } = useAgGridHelpers(
-    isNewTheme ? "ag-theme-odyssey" : undefined
+    isNewTheme ? "ag-theme-salt" : undefined
```

Comment:
I'd personally rename 'isNewTheme' to 'isSaltTheme'

Reply:
good idea, i agree - will update it now. Might as well